docker/build-tools/Dockerfile

```diff
@@ -75,6 +75,7 @@ ENV SHELLCHECK_VERSION=v0.9.0
 ENV SU_EXEC_VERSION=0.3.1
 ENV TRIVY_VERSION=0.46.0
 ENV YQ_VERSION=4.35.2
+ENV OC_VERSION=4.13.19
```

Comment:
The ENV variables are listed alphabetically, so this line should be moved up.

Reply:
Done. I also added a comment to help keep the list sorted